tests/Unit/Infrastructure/EudonetParis/EudonetParisTransformerTest.php

```diff
@@ -79,7 +79,7 @@ public function testTransform(): void
 
         $generalInfoCommand = new SaveRegulationGeneralInfoCommand();
         $generalInfoCommand->identifier = '20230514-1';
-        $generalInfoCommand->category = RegulationOrderCategoryEnum::OTHER->value;
+        $generalInfoCommand->subject = RegulationSubjectEnum::OTHER->value;
```

Comment:
Attention ici la category n'est plus configurée

```suggestion
        $generalInfoCommand->category = RegulationCategoryEnum::TEMPORARY_REGULATION->VALUE;
        $generalInfoCommand->subject = RegulationSubjectEnum::OTHER->value;
```